msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -0,0 +1,158 @@
+
+This article covers information about the meeting lifecycle and how to integrate tabs, bots, and messaging extensions in your meeting. It also enables you to identify participant roles and also use different user types to perform tasks.
+
+> [!NOTE]
+> To work with the meeting app extensibility features, you must have the appropriate permissions.
+
+## Meeting lifecycle
+
+Meeting lifecycle consists of pre-meeting, in-meeting, and post-meeting app experience. You can integrate tabs, bots, and messaging extensions in each stage of the meeting lifecycle.
+
+### Integrate tabs into the meeting lifecycle
+
+Tabs allow team members to access services and content in a specific space within a channel or chat. This lets the team work directly with tabs and have conversations about the tools and data available within tabs. In Teams meeting, users can add a tab by selecting plus <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>, and choosing the app that they want to install as a tab.
+
+> [!IMPORTANT]
+> If you have integrated a tab with your meeting, then your app must follow the Teams [single sign-on (SSO) authentication flow](../tabs/how-to/authentication/auth-aad-sso.md) for tabs.
+
+> [!NOTE]
+> * Mobile clients support tabs only in pre and post meeting stages. The in-meeting experiences that is in-meeting dialog and panel are currently not available on mobile.
+> * Apps are supported only in private scheduled meetings.
+
+#### Pre-meeting app experience
+
+With the pre-meeting app experience, you can find and add meeting apps and perform pre-meeting tasks, such as developing a poll to survey meeting participants.
+
+**To add tabs to an existing meeting**
+
+1. In your calendar, select a meeting to which you want to add a tab.
+1. Select the **Details** tab and select plus <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>. The tab gallery appears.
+
+    ![Pre-meeting experience](../assets/images/apps-in-meetings/PreMeeting.png)
+
+1. In the tab gallery, select the app that you want to add and follow the steps as required. The app is installed as a tab.
+
+    ![Pre-meeting tab view](../assets/images/apps-in-meetings/PreMeetingTab.png)
+
+    > [!NOTE]
+    > * You can also add a tab using the meeting **Chat** tab in an existing meeting.
+    > * Tab layout must be in an organized state if there are more than ten polls or surveys.
+
+#### In-meeting app experience
+
+With the in-meeting app experience, you can engage participants during the meeting by using apps or taking polls. Meeting apps are hosted in the top upper bar of the chat window as an in-meeting tab. When users add a tab to a meeting through the tab gallery, apps that are in-meeting experiences are displayed. Use the in-meeting dialog box to showcase actionable content for meeting participants. For more information, see [create apps for Teams meetings](create-apps-for-teams-meetings.md).
```

Comment:
Reword: 

With the in-meeting app experience, you can engage participants during the meeting by using apps and the in-meeting dialog. Meeting apps are hosted in the top upper bar of the meeting window as an in-meeting tab. Use the in-meeting dialog box to showcase actionable content for meeting participants. For more information, see [create apps for Teams meetings](create-apps-for-teams-meetings.md).